packages/relay/src/lib/db/repositories/hbarLimiter/hbarSpendingPlanRepository.ts

```diff
@@ -175,6 +175,34 @@ export class HbarSpendingPlanRepository {
     }
   }
 
+  /**
+   * Finds all active hbar spending plans by subscription type.
+   * @param {SubscriptionType} subscriptionType - The subscription type to filter by.
+   * @returns {Promise<IDetailedHbarSpendingPlan[]>} - A promise that resolves with the active spending plans.
+   */
+  async findAllActiveBySubscriptionType(subscriptionType: SubscriptionType): Promise<IDetailedHbarSpendingPlan[]> {
+    const callerMethod = this.findAllActiveBySubscriptionType.name;
+    const keys = await this.cache.keys(`${this.collectionKey}:*`, callerMethod);
```

Comment:
What's the difference between using `findAllActiveBySubscriptionType` directly and defining a constant using the method name? In both cases, if the method name is changed, we must edit the method's body as well. To make the function dynamic, we can use `arguments.callee.name` but honestly I don't know what advantage we're gaining.

Reply:
The IDE will automatically rename this constant when the method name is "refactored" (as it changes all code occurrences of the method when renaming it)

If we set this to a hardcoded string, we should explicitly tell the IDE to also "rename all text occurrences" of the method in addition to code occurrences, and it's easier to be missed.